app/frontend/src/AppRoute.tsx

```diff
@@ -23,17 +24,21 @@ export const useStyles = makeStyles((theme) => ({
     paddingLeft: theme.spacing(2),
     paddingRight: theme.spacing(2),
   },
+  fullWidthContainer: {
+    paddingLeft: 0,
+    paddingRight: 0,
+  },
 }));
 
 interface AppRouteProps extends RouteProps {
   isPrivate: boolean;
-  isFullscreen?: boolean;
+  variant?: "standard" | "full-screen" | "full-width";
```

Comment:
Switched to using `variant` here as booleans aren't really a good idea when there are more than two state values, since boolean combos quickly introduce impossible states and makes things hard to reason about.